src/main/java/com/faforever/client/preferences/ChatPrefs.java

```diff
@@ -64,6 +66,18 @@ public void setTimeFormat(TimeInfo time) {
     this.timeFormat.set(time);
   }
 
+  public ChatFormat getChatFormat() {
+    return this.chatFormat.get();
+  }
+
+  public void setChatFormat(ChatFormat chatFormat) {
+    this.chatFormat.setValue(chatFormat);
+  }
+
+  public ObjectProperty<ChatFormat> getChatFormatProperty() {
+    return chatFormat;
+  }
```

Comment:
the getter for properties is just called "chatFormatProperty()" throughout the project afaik. (have a look at the method below)

Reply:
fixed:)